lib/core/src/Cardano/Wallet/Primitive/Types/Tx.hs

```diff
@@ -0,0 +1,340 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.Wallet.Primitive.Types.Tx
```

Comment:
You have doc comments for the other new modules, but not in this one

Reply:
> You have doc comments for the other new modules, but not in this one

Great point @Anviking. I'll add this in a follow-up PR.